pkgs/pinned.nix

```diff
@@ -1,13 +1,16 @@
 let
   nixpkgsPinned = import ./nixpkgs-pinned.nix;
-  unstable = import nixpkgsPinned.nixpkgs-unstable { config = {}; overlays = []; };
-  nixBitcoinPkgsUnstable = import ./. { pkgs = unstable; };
+  nixpkgs-unstable = import nixpkgsPinned.nixpkgs-unstable { config = {}; overlays = []; };
+  nixpkgs-stable = import nixpkgsPinned.nixpkgs { config = {}; overlays = []; };
 in
-{
+rec {
   inherit (unstable)
```

Comment:
These must be inherited from unstable nixpkgs, not the unstable nix-bitcoin packages, which don't include, e.g., bitcoind.

Reply:
you are correct, not sure what I was smoking when I did this PR